### PR TITLE
[FLINK-6941][table]Add validate that the start and end window propert…

### PR DESCRIPTION
**Pull request by sunjincheng121**
Add validate that the start and end window properties are not accessed on over windows.
- [x] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [x] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala [hunk 1]
```diff
@@ -1004,6 +1004,11 @@ class OverWindowedTable(
       table.logicalPlan,
       table.tableEnv)
 
+    if(fields.exists(_.isInstanceOf[WindowProperty])){
+      throw ValidationException(
+        "Window start and Window end can not be used on over windowed tables.")
```
Comment (fhueske) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala]: What do you think about `"Window start and end properties are not available for Over windows."`?

Comment (sunjincheng121) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala]: Also make sense to me.

**Comment by sunjincheng121** [comment]
Hi @fhueske thanks for your reviewing. I have updated the PR according your suggestion.

Best,
SunJincheng

**Comment by fhueske** [comment]
thanks! 
+1 to merge

**Comment by fhueske** [comment]
will merge this

**Pull request closed without merging**